server/middlewares/statsRedis.js

```diff
@@ -49,6 +49,8 @@ const trackTxSubmissions = mung.json((body, req, res) => {
 
     const txWalletType = req.get('walletType')
     incrCountersBy(`${txSubmissionType}:${txWalletType}`, 1)
+    const txCryptoProviderVersion = req.get('cryptoProviderVersion')
+    incrCountersBy(`walletVersions:${txWalletType}-${txCryptoProviderVersion}`, 1)
```

Comment:
if I understand correctly, this way we only track the absolute number of tx submits by wallet version. This isn't really useful as we need to know how the version distribution changes over time to have the insight what's the adoption of newer versions of firmwares as they are rolled out

In general, I think redis is already too polluted and I see it as a place for the most critical business metrics which hw wallet version really isn't. So I'd rather leverage the google analytics stats for that which are not that used but if we need info, it's there and GA also takes care of distributing the data over time without us explicitly caring.

Reply:
Actually, this would track the used versions totally, monthly and daily so we would be able to track adoption. Anyway I added the support for google analytics, I ll create a dashboard for it once its merged.

Reply:
Btw. It makes sense to drop the middle commit ( redis stats ).